proposals/description/tip_vebalotc.ts

```diff
@@ -62,23 +62,30 @@ const tip_vebalotc: TemplatedProposalDescription = {
       target: 'proxyAdmin',
       values: '0',
       method: 'upgrade(address,address)',
-      arguments: (addresses) => [addresses.balancerGaugeStaker, addresses.veBoostManagerImplementation],
+      arguments: (addresses) => [addresses.balancerGaugeStakerProxy, addresses.balancerGaugeStakerV2Impl],
       description: `Upgrade implementation of the balancerGaugeStaker`
     },
     {
-      target: 'veBoostManager',
+      target: 'balancerGaugeStaker',
       values: '0',
       method: '_initialize(address,address)',
       arguments: (addresses) => [addresses.vebalOtcHelper, addresses.balancerVotingEscrowDelegation],
-      description: 'Initialize veBoostManagerProxy state variables'
+      description: 'Initialize BalancerGaugeStakerProxy state variables'
     },
+    // {
+    //   target: 'balancerGaugeStakerV2Impl',
+    //   values: '0',
```

Comment:
Why is this commented out?

Reply:
The initialization of the implementation was failing for some reason (think its related to how the script work). Need to carefully review it again

Reply:
It is failing expectedbly. You are calling the initialise() function of the V1 implementation contract which has already been initialised and should not be called again. 

You should be calling the initialise() function of the V2 implementation contract

Reply:
Ignore - failing due to another reason